config/default.json5

```diff
@@ -66,7 +66,6 @@
     relayHubAddress: '0x0000000000000000000000000000000000000000', // relay hub contract address, you can retrieve this from the contract summary.
     relayVerifierAddress: '0x0000000000000000000000000000000000000000', // relay verifier contract address, you can retrieve this from the contract summary.
     deployVerifierAddress: '0x0000000000000000000000000000000000000000', // deploy verifier contract address, you can retrieve this from the contract summary.
-    smartWalletFactoryAddress: '0x0000000000000000000000000000000000000000',
```

Comment:
do we not need this address then?

Reply:
not its not necessary, based on the functions that are used by the server, its no longer necessary to include it now.